packages/fxa-settings/src/components/Profile/index.test.tsx

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { Profile } from '.';
+import {
+  renderWithRouter,
+  MockedCache,
+  MOCK_ACCOUNT,
+} from '../../models/_mocks';
+
+// set this here to bypase ts null warning
+let displayName = 'John Dope';
+if (MOCK_ACCOUNT.displayName) {
+  displayName = MOCK_ACCOUNT.displayName;
+}
```

Comment:
I believe you can do: `const displayName = MOCK_ACCOUNT.displayName!` to tell TS it can't be `null`.